src/components/ResultsListItem.vue

```diff
@@ -1,14 +1,22 @@
 <template>
   <section class="item-card">
-    <button @click="savePark" class="save-button">
-      SAVE
-    </button>
+    <article class='buttons-article'>
+      <button @click="savePark" class="save-button">
+        SAVE
+      </button>
+      <router-link :to="`/results/${result.name}`" >
+        <button class='details-button'>
```

Comment:
Thanks for adding this `details` button that dynamically routes!

Reply:
I'm a big fan of an App having multiple ways to get to an endpoint, as this makes it easier for a user to get where they want to go without having to navigate back to a specific place in order to get there. Perhaps we can incorporate this in multiple areas.